qa-tests-backend/src/main/groovy/services/BaseService.groovy

```diff
@@ -137,6 +139,8 @@ class BaseService {
                     // .enableRetry()
                     .negotiationType(NegotiationType.TLS)
                     .sslContext(sslContext)
+                    .keepAliveTime(1, TimeUnit.SECONDS)
+                    .idleTimeout(1, TimeUnit.MINUTES)
```

Comment:
This tells to send a keep-alive ping in one second of inactivity, and shuts down an inactive connection after 1 minute.
Not sure if that will restore a remotely closed connection.

Reply:
At least we will learn about this with keep alive:
> Use keepalive to notice connection failures while RPCs are in progress. Use idleTimeout to release resources and prevent idle TCP connections from breaking when the channel is unused.

Reply:
here is a nice overview https://grpc.io/docs/guides/keepalive/
keepalice is for active for idle it's `keepAliveWithoutCalls`

Reply:
Maybe we can lower idle timeout and not use keep alive at all